src/main/java/lotto/domain/Result.java

```diff
@@ -1,21 +1,17 @@
 package lotto.domain;
 
-import java.util.*;
-import java.util.function.Function;
-import java.util.stream.Collectors;
+import java.util.Arrays;
+import java.util.List;
 
 public enum Result {
 
     FIRST(6, 2_000_000_000, " "),
     SECOND(5, 30_000_000, ", 보너스 볼 일치"),
     THIRD(5, 1_500_000, " "),
-    FOURTH(4, 50_000," "),
+    FOURTH(4, 50_000, " "),
```

Comment:
`Result`에서 `bonus`는 view에 해당하는 역할을 가지네요
아래와 같이 bonus를 빼고 구현해보는건 어떨까요?
```suggestion
    FOURTH(4, 50_000),
```

Reply:
`Result`에서 bonus를 없애고 구현하려고 하는데 마땅히 떠오르는건 삼항연산자 밖에 생각이 안나네요ㅜ
우테코에서 삼항연산자는 지양하라고 하는걸로 아는데 더 좋은 방법이 있을까요?